web-app/src/main/webapp/css/style.css

```diff
@@ -23,3 +24,28 @@
   font-size: 2em;
   height: 3em;
 }
+
+/* Report form */
+#report-form {
+  position: absolute;
+  top: 200px;
+  z-index: 3;
+}
+
+/* Bottom dock */
+#report-button {
+  bottom: 200px;
+  height: 200px;
+  position: absolute;
+  width: 200px;
+  z-index: 3;
+}
+
+#dock {
```

Comment:
Not a big deal, super nit: Could consider using material design terminology: https://material.io/components/app-bars-bottom#usage